DESCRIPTION

```diff
@@ -12,7 +12,8 @@ Authors@R: c(
   person("Eduard","Antonyan", role="ctb"),
   person("Markus","Bonsch",   role="ctb"),
   person("Hugh","Parsonage",  role="ctb"),
-  person("Scott","Ritchie",   role="ctb"))
+  person("Scott","Ritchie",   role="ctb"),
+  person("Mus","Yaramaz-David", role="ctb"))
```

Comment:
People don't normally add themselves here. There are 53 contributors who have done something, no matter how small:  https://github.com/Rdatatable/data.table/graphs/contributors. You would be listed there, and in NEWS. When a contributor has made sustained contributions, or a few big ones, I add them to DESCRIPTION which is displayed on CRAN. The contributors listed in DESCRIPTION are roughly listed at the top of https://github.com/Rdatatable/data.table/graphs/contributors.
At least, this is the way this has been done to date.  We could change so that every single contributor is listed in DESCRIPTION if people want to.

Reply:
I think that something mid-way between the two would be ideal.

For example, those who make small, non-functional changes (such as correcting typos) wouldn't be included here but those who extend the functionality of `data.table` in some meaningful and/or useful way should be included.

That way it would prevent this file from becoming too large, whilst still retaining its relevance and meaning.

Reply:
I think the author field should be limited to those who have contributed code that the authors would not have ordinarily come up with. This PR is an enhancement but its implementation is quite elementary so should not qualify.